data/companies.json

```diff
@@ -368,6 +368,22 @@
         "youtube": "https://www.youtube.com/@Hamropatro"
       }
     },
+    {
+      "name": "HazeSoft Pvt. Ltd.",
+      "establishedAt": 1350227551,
```

Comment:
What is the source for this?

Reply:
![image](https://github.com/Know-Nepal/tech-companies/assets/64427471/ad14a7de-2073-41fc-9f44-8415769bc140)